<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDock</class>
 <widget class="QDockWidget" name="SearchDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recherche</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="gpbxSearchFilter">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string>Filtre de recherche</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="3" column="0">
        <widget class="QCheckBox" name="chkProjectName">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Nom du projet</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="chkContributory">
         <property name="text">
          <string>Nom de la prestation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="chkReferentName">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Nom du référent</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chkSearchCompany">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Nom de la société</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="chkBillQuote">
         <property name="text">
          <string>N°,Nom du devis/facture</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="leSearch">
      <property name="styleSheet">
       <string notr="true">background: url(:/icons/searchMini);
background-position: right;
background-repeat: no-repeat;     
border-radius: 0px;
height: 23px;
border: 1px solid #bbb;</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Effectuer une recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leSearch</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SearchDock</receiver>
   <slot>search(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search(QString)</slot>
 </slots>
</ui>
